net/host.go

```diff
@@ -181,42 +169,24 @@ func NewHost(cfg *Config) (*host, error) {
 	hst := &host{
 		ctx:        ourCtx,
 		cancel:     cancel,
-		protocolID: fmt.Sprintf("%s/%s/%s/%d", protocolID, protocolVersion, cfg.Environment, cfg.EthChainID),
+		protocolID: fmt.Sprintf("%s/%s/%d", cfg.ProtocolID, cfg.Environment, cfg.EthChainID),
```

Comment:
How are we preventing hosts with incompatible API versions from talking to each other?

Reply:
Answer: The version is now part of the ProtocolID.